From 3888c829511b97d08462844312f24825dea413d9 Mon Sep 17 00:00:00 2001
From: wangzx <593074943@qq.com>
Date: Tue, 20 Apr 2021 22:33:26 +0800
Subject: [PATCH] dwc_otg: fix an undeclared variable Replace an
 undeclared variable used by DWC_DEBUGPL with the real endpoint address.
 DWC_DEBUGPL does nothing with DEBUG undefined so it did not go wrong before.
 Signed-off-by: Zixuan Wang <wangzixuan@sjtu.edu.cn>

---
 drivers/usb/host/dwc_otg/dwc_otg_hcd_linux.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/usb/host/dwc_otg/dwc_otg_hcd_linux.c
+++ b/drivers/usb/host/dwc_otg/dwc_otg_hcd_linux.c
@@ -1026,7 +1026,8 @@ static void endpoint_reset(struct usb_hc
 	dwc_irqflags_t flags;
 	dwc_otg_hcd_t *dwc_otg_hcd = hcd_to_dwc_otg_hcd(hcd);
 
-	DWC_DEBUGPL(DBG_HCD, "DWC OTG HCD EP RESET: Endpoint Num=0x%02d\n", epnum);
+	DWC_DEBUGPL(DBG_HCD, "DWC OTG HCD EP RESET: Endpoint Num=0x%02d\n",
+		    ep->desc.bEndpointAddress);
 
 	DWC_SPINLOCK_IRQSAVE(dwc_otg_hcd->lock, &flags);
 	if (ep->hcpriv) {
